src/general/disallowanonymous/constraint.tmpl

```diff
@@ -24,6 +24,11 @@ spec:
               type: array
               items:
                 type: string
+            disallowAuthenticated:
```

Comment:
Optionally you could also include `default: false`, while also implemented in the Rego, makes it clearer to users the default behavior.

Reply:
Good call, included.